sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -183,7 +188,7 @@ namespace Azure { namespace Core { namespace Http {
      * If it is found, it will insert query parameters to m_queryParameters internal field
      * and remove it from url
      */
-    const std::string parseUrl(std::string const& url)
+    const std::string saveAndRemoveQueryParameter(std::string const& url)
```

Comment:
do we write const on the left, or on the right? return type is left, argument is on the right. I suggest to be consistent, and I personally prefer on the right - there are benefits. We did right in the C repo.

Reply:
weak suggestion

Reply:
+1 for being consistent, but I prefer on the left.